File: app/src/main/java/com/example/vivic/nolost/bean/Goods.java
package com.example.vivic.nolost.bean;

import java.util.List;

import cn.bmob.v3.BmobObject;
import cn.bmob.v3.datatype.BmobFile;

public class Goods extends BmobObject {
    public static final String TYPE_LOST = "丢失";
    public static final String TYPE_FOUND = "拾获";

    public String name;
    public String detail;
    public List<BmobFile> photoList;
    public String location;
    public MyUser createor;
    public String type;
}

File: app/src/main/java/com/example/vivic/nolost/bean/Goods.kt
package com.example.vivic.nolost.bean

import cn.bmob.v3.BmobObject

class Goods : BmobObject() {

    var name: String? = null
    var location: String? = null
    var detail: String? = null
    var photoList: List<String>? = null
    var type: String? = null
    var creatorObjectId: String? = null
    var creatorName: String? = null
    var creatorAvatar: String? = null


    companion object {
        const val TYPE_ALL = ""
        const val TYPE_LOST = "lost"
        const val TYPE_FOUND = "found"
    }

    override fun toString(): String {
        return "\n Goods:" +
                "type:$type" +
                ",name:$name" +
                ",location:$location" +
                ",detail:$detail" +
                ",photoList:$photoList" +
                "\ncreatorObjectId:${creatorObjectId}creatorName:$creatorName,creatorAvatar:$creatorAvatar \n"
    }
}
